internal/xpkg/v2/build.go

```diff
@@ -1,261 +0,0 @@
-/*
```

Comment:
very cool to already clean up the `xpkg/v2` temp package also - you're getting a ton of things clean in this PR @negz!! 🧹 🗑️